src/style.css

```diff
@@ -1 +1,115 @@
-/* add styles here */
+.html {
+  margin: 0;
+  padding: 0;
+}
+
+@font-face {
+  font-family: Avenir;
+  src:
+    url(./fonts/Avenir-Book.ttf) format('Avenir'),
+    url(./fonts/Avenir-Heavy.ttf) format('Avenir'),
+    url(./fonts/Avenir.ttc);
+}
+
+input {
+  border: none;
+  outline: 0;
+}
+
+input::placeholder {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
```

Comment:
The `font-family` should include a fallback font, in case the primary font isn't available. For example, `font-family: Avenir, sans-serif;`.